031-custom_authentication.md

```diff
@@ -0,0 +1,200 @@
+# Custom Authentication
+
+This proposal focuses on supporting custom authentication for SASL/mTLS in the Strimzi operator. 
+
+## Current situation
+
+Currently, numerous authentication methods are [supported](https://github.com/strimzi/strimzi-kafka-operator/tree/0.25.0/api/src/main/java/io/strimzi/api/kafka/model/authentication) in Strimzi, and allows the user to configure which one to use. In addition, in the case a custom *authorizer* needs to be used, this is also currently [supported](https://github.com/strimzi/strimzi-kafka-operator/blob/0.25.0/cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBrokerConfigurationBuilder.java#L541-L549).
+
+However, there is no ability to specify custom authentication, which is what this proposal will focus on.
```

Comment:
Again, just for clarity ... maybe you could add that it focuses only on the broker part.